client/app/components/Accordion/__tests__/Accordion.spec.jsx

```diff
@@ -45,4 +48,25 @@ describe('Accordion', () => {
       expect(accordionContent).toHaveClass('accordionContent');
     });
   });
+
+  describe('open when enter key is pressed', () => {
+    it('toggles correctly', () => {
+      const component = (
+        <Accordion id={id} title={title} onKeyDown={openToggle}>
+          {children}
+        </Accordion>
+      );
+      const { getByRole } = render(component);
+
+      const accordionContent = getByRole('region');
+      const accordionBtn = getByRole('button');
+      fireEvent.keyDown(accordionContent, {
```

Comment:
You want to trigger the `Enter` key on the `accordionBtn` not `accordionContent`. After doing that, `expect(accordionContent).toHaveClass('accordionContent');` should pass. `expect(accordionContent).toHaveClass('accordionClose');` should only pass once the accordion is closed.

Reply:
Sorry, @julianguyen I haven't written any test cases before that's why it is taking some time. Hopefully, it's completed now.